### PR TITLE
Merging note and entity

### PR DESCRIPTION
**Pull request by Mati20041**

### CODE REVIEW

File: src/app/notes/[noteId]/NoteBox.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+'use client';
+import { Note } from '@prisma/client';
+import { toast } from 'react-toastify';
+import { updateNote } from '@/app/notes/notesActions';
```
Comment (miloszmazur) [inline src/app/notes/[noteId]/NoteBox.tsx]: 👏 good pattern

File: src/app/notes/[noteId]/page.tsx [hunk 2]
```diff
@@ -1,23 +1,13 @@
...
-        defaultValue={note.title}
-      />
-      <input
-        id="text"
-        type="text"
-        className="m-3 bg-transparent w-full"
-        defaultValue={note.text}
-      />
-    </form>
+    <div>
```
Comment (miloszmazur) [inline src/app/notes/[noteId]/page.tsx]: nit: why do we surround `<form>` with a `<div>`? And if we need to, what do you think of making it a part of `NoteBox`?

File: src/app/notes/notesActions.ts
Comment (miloszmazur) [inline src/app/notes/notesActions.ts:1]: okay, hear me out: how about just "actions.tsx"? More next-y

File: src/app/notes/notesList/NotesList.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { CreateNewNote } from '@/app/notes/notesList/CreateNewNote';
+import { NoteItem } from '@/app/notes/notesList/NoteItem';
+
+export const NotesList: React.FunctionComponent = async () => {
+  const notes = await getNotes();
+
+  return (
+    <div>
+      <h2>notes list:</h2>
+      <div>
```
Comment (miloszmazur) [inline src/app/notes/notesList/NotesList.tsx]: same pattern w/ `<divs>` here too

File: src/app/notes/notesActions.ts
Comment (Mati20041) [reply src/app/notes/notesActions.ts:1]: I'm afraid if it wouldn't be too generic. It doesn't say much about what kind of actions, what if we would like to also add "linksActions.ts" etc?

File: src/app/notes/[noteId]/page.tsx (hunk 2 above)
Comment (Mati20041) [reply src/app/notes/[noteId]/page.tsx]: for no reason 🤡

File: src/app/notes/notesActions.ts
Comment (miloszmazur) [reply src/app/notes/notesActions.ts:1]: yeah pretty much. `actions.tsx` will keep server actions for a given route

**Pull request merged**